little_timmy/config_loader.py

```diff
@@ -27,6 +27,105 @@
 
 LOGGER = logging.getLogger("little-timmy")
 
+
+def find_and_setup_galaxy_collections(root_dir: str, skip_dirs: list[str]) -> list[str]:
+    """
+    Find all galaxy collections in the root directory and set them up for FQDN access.
+    Creates in-memory Python modules without modifying the filesystem.
+    Returns empty list (kept for compatibility).
+    """
+    import sys
+    import types
+    from glob import iglob
+    
+    # Look for galaxy.yml files which indicate a collection
+    for galaxy_file in iglob(f"{root_dir}/**/galaxy.yml", recursive=True):
+        # Skip if in excluded directories
+        relative_path = os.path.relpath(os.path.dirname(galaxy_file), root_dir)
+        if any(excluded_dir in relative_path for excluded_dir in skip_dirs):
+            continue
+            
+        collection_dir = os.path.dirname(galaxy_file)
+        
+        # Read galaxy.yml to get namespace and name
+        try:
+            with open(galaxy_file, 'r') as f:
+                galaxy_info = yaml.safe_load(f)
+                
+            namespace = galaxy_info.get('namespace')
+            name = galaxy_info.get('name')
+            
+            if namespace and name:
+                # Check if collection is already in the correct structure
+                expected_path = f"ansible_collections/{namespace}/{name}"
+                if collection_dir.endswith(expected_path):
+                    # Already in correct structure, add the parent directory to sys.path
+                    parent_dir = collection_dir[:-len(expected_path)].rstrip('/')
+                    if parent_dir and parent_dir not in sys.path:
+                        sys.path.insert(0, parent_dir)
```

Comment:
if the dir structure is correct, do we need to do anything? Won't ansible load the module correctly by itself?